<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Depth Enhancement by means of Data Fusion - http://www.frederic-garcia-becerro.com - frederic.garcia.becerro@gmail.com</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="ui_gb_FilterSettings">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>211</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Filter Settings</string>
    </property>
    <widget class="QSlider" name="ui_sl_sigmaSpatial">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>171</width>
       <height>19</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ui_lbl_sigmaSpatial">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Sigma Spatial: 10</string>
     </property>
    </widget>
    <widget class="QLabel" name="ui_lbl_sigmaRange">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Sigma Range: 10</string>
     </property>
    </widget>
    <widget class="QSlider" name="ui_sl_sigmaRange">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>171</width>
       <height>19</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ui_lbl_sigmaCredMap">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Sigma Credibility Map: 10</string>
     </property>
    </widget>
    <widget class="QSlider" name="ui_sl_sigmaCredMap">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>171</width>
       <height>19</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="sliderPosition">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="ui_cb_dataFusion_Filter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>RGB-D Filter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UML Filter (RGB)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UML Filter (Grayscale)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PWAS Filter (RGB)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PWAS Filter (Grayscale)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>JBU Filter (Grayscale)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bilateral Filter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-------------------------------------------</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>JBU Filter (Kopf et al.)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NJBU Filter (Kim et al.)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NAFDU Filter (Chan et al.)</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="ui_pb_ApplyFilter">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Apply Filter</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ui_gb_Display">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>481</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Display</string>
    </property>
    <widget class="QComboBox" name="ui_cb_dataFusion_Display">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Guid. Img, I_RGB</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Guid. Img, I</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DS Guid. Img, I_ds</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Depth Map, D</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DS Depth Map, D_ds</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Credibility Map, Q</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DS Credibility Map, Q_ds</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Blending Mask, B</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Enhanced Depth Map, J</string>
      </property>
     </item>
    </widget>
    <widget class="QSlider" name="ui_sl_minPalette">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>171</width>
       <height>19</height>
      </rect>
     </property>
     <property name="maximum">
      <number>750</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ui_lbl_minPalette">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Min palette: 0 mm</string>
     </property>
    </widget>
    <widget class="QSlider" name="ui_sl_maxPalette">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>171</width>
       <height>19</height>
      </rect>
     </property>
     <property name="maximum">
      <number>750</number>
     </property>
     <property name="value">
      <number>350</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ui_lbl_maxPalette">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Max palette: 3500 mm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_pb_SaveDisplayedImage">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Displayed Image</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="ui_lbl_dataFusion_smallWin_Depth">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>75</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>75</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Depth Map</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ui_lbl_dataFusion_smallWin_RGB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>75</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>75</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Guidance Image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ui_lbl_dataFusion_mainWin">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>480</width>
      <height>360</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>240</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ui_pb_LoadDepthMap">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Depth Map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ui_pb_LoadGuidImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Guid. Image</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>20</width>
      <height>481</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Guid_Image_I"/>
    <addaction name="actionLoad_Depth_Map_D"/>
    <addaction name="actionSave_Displayed_Image"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLoad_Guid_Image_I">
   <property name="text">
    <string>Load Guid. Image, I</string>
   </property>
  </action>
  <action name="actionLoad_Depth_Map_D">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load Depth Map, D</string>
   </property>
  </action>
  <action name="actionSave_Displayed_Image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Displayed Image</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
